### ISSUE TITLE
Dropdown does not take focus when opens w/ iOS + VO

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#764]
### Expected Behaviour
opening the dropdown with iOS & VO focus should move to the dropdown entries.

### Actual Behaviour
Focus remains on the trigger when the dropdown opens. swiping right moves to the next element in the page not the dropdown entries

### Reproduce Scenario (including but not limited to)
https://opensource.adobe.com/spectrum-web-components/components/dropdown/examples

#### Steps to Reproduce
- https://opensource.adobe.com/spectrum-web-components/components/dropdown/examples
- Turn on VO
- focus onto the dropdown
- double tap to open the dropdown
on iPadOS 14 the dropdown opens but focus does not move to the choices in the dropdown.
on iOS 13 the dropdown does not even open.

#### Platform and Version
iPadOS14 - Safari - VO
iOS13 - Safari - VO

**Comment by Westbrook** [comment]
I've confirmed this locally. We'll need to track down what event/interaction is being triggered by the "double-tap" interaction and ensure that it also throws focus for this and for other "replace" type overlays.

**Comment by Westbrook** [comment]
Is this a timing situation?

**Comment by Westbrook** [comment]
Spent a bunch of time this week reviewing the issues here, and learned a lot about the intricacies of this interaction.

Findings:
- role values + focus not enough to associate the UI elements together
- consume/apply IDs for each item
- focus the listbox not the individual items
- synthesize item focus with JS
- manage `aria-activedecendant` and `aria-selected`
- translate listbox click to click on "active" item

The changes cross a handful of elements, but should be well within reach for correcting this issue (which occurs across all VO contexts).
